src/main/java/JavaReferenceDataTypes/DescribeOutput.java

```diff
@@ -0,0 +1,32 @@
+package JavaReferenceDataTypes;
+
+public class DescribeOutput {
+    public static int[] make(int n) {
+        int[] a = new int[n];
+        for (int i = 0; i < n; i++) {
+            a[i] = i + 1; // completeaza array-ul(cu make.length = 5) cu valori, atribuind ca valoare = indexul + 1
```

Comment:
Usually we comment the entire method - not statements.

Reply:
Done